destination/destination_test.go

```diff
@@ -0,0 +1,142 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package destination
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/conduitio-labs/conduit-connector-clickhouse/config"
+	"github.com/conduitio-labs/conduit-connector-clickhouse/destination/mock"
+	sdk "github.com/conduitio/conduit-connector-sdk"
+	"github.com/golang/mock/gomock"
+	"github.com/matryer/is"
+)
+
+func TestDestination_ConfigureSuccess(t *testing.T) {
+	t.Parallel()
+
+	is := is.New(t)
+
+	d := Destination{}
+
+	err := d.Configure(context.Background(), map[string]string{
+		config.URL:            "localhost:8123?username=user&password=password&database=default",
+		config.Table:          "test_table",
+		config.PrimaryColumns: "id",
+	})
+	is.NoErr(err)
+	is.Equal(d.cfg, config.General{
+		URL:            "localhost:8123?username=user&password=password&database=default",
+		Table:          "test_table",
+		PrimaryColumns: []string{"id"},
+	})
+}
+
+func TestDestination_ConfigureFail(t *testing.T) {
+	t.Parallel()
+
+	is := is.New(t)
+
+	d := Destination{}
```

Comment:
please change Destination{} to NewDestination().

Reply:
The function `NewDestination` returns the interface, but I have to check the field `cfg` of the `Destination` struct.

Reply:
you don't check the cfg field on this test

Reply:
Ok, done.